modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Bicrosswalk.kt

```diff
@@ -0,0 +1,53 @@
+package arrow.typeclasses
+
+import arrow.Kind
+import arrow.Kind2
+import arrow.core.identity
+
+interface Bicrosswalk<T> : Bifunctor<T>, Bifoldable<T> {
```

Comment:
"Unfortunately it is too perfect an abstraction to be useful. - elliott" I love this quote :D
https://teh.id.au/posts/2017/03/29/these-align-crosswalk/

I wonder when the first time comes around where I actually have a legitimate use of this typeclass :)
Either way it's not wrong having it anyway.

Reply:
yes, i also came across that article :-D